src/mono/wasm/debugger/BrowserDebugProxy/MemberObjectsExplorer.cs

```diff
@@ -331,7 +332,18 @@ JArray GetHiddenElement()
 
                 typePropertiesBrowsableInfo.TryGetValue(propName, out DebuggerBrowsableState? state);
 
-                if (allMembers.TryGetValue(propName, out JObject backingField))
+                // handle parents' members:
+                var propNameWithSufix = propName;
+                JObject backingField;
+                if (allMembers.TryGetValue(propName, out backingField)
+                    && !isOwn
```

Comment:
Move `isOwn` to be the first condition, so if it is true then we avoid doing rest of the stuff.

Reply:
But then we need backingField value anyway (if isOwn: true). So it made for me more sense not to TryGetValue twice, we would have to call it anyway.